CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## v1.12.1
+
+* Bug - [Cleanup pod networking resources when IPAMD is unreachable to prevent rule leaking.](https://github.com/aws/amazon-vpc-cni-k8s/pull/2145) (@jdn5126 )
```

Comment:
Minor nit: let's move the "Bug" below line 15.

Reply:
alphabetized in latest PR